ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/stun.j2

```diff
@@ -0,0 +1,24 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - stun #}
+{% if stun.server is arista.avd.defined %}
+stun
+   server
+{%     if stun.server.local_interface is arista.avd.defined %}
+      local-interface {{ stun.server.local_interface }}
+{%     endif %}
+{%     if stun.client is arista.avd.defined %}
+client
+{%         for server in stun.client %}
+{%             if server.server_profile is arista.avd.defined %}
+   server_profile {{ server.server_profile }}
+{%             endif %}
+{%             if server.ip_address is arista.avd.defined %}
+      ip address {{ server.ip_address }}
+{%             endif %}
+{%         endfor %}
+{%     endif %}
```

Comment:
This jinja2 template is not matching your schema + there is some wrong indentation

```suggestion
{%     if stun.client is arista.avd.defined %}
   client
{%         for profile in stun.client.server_profiles | arista.avd,default([]) %}
      server_profile {{ profile.name }}
{%             if server.ip_address is arista.avd.defined %}
         ip address {{ profile.ip_address }}
{%             endif %}
{%         endfor %}
{%     endif %}
```

Please check correct indent in generated config as this is "freely handtyped" in github markdown so it may need some adjustement :)